pkg/cluster/cluster.go

```diff
@@ -367,6 +367,6 @@ func stringSliceToByteSliceSlice(ss []string) [][]byte {
 }
 
 // ListNodes returns the list of container IDs for the "nodes" in the cluster
-func (c *Context) ListNodes() (containerIDs []string, err error) {
+func (c *Context) ListNodes() ([]*nodes.Node, error) {
```

Comment:
(nit) is it useful/does it make sense for this to be a slice of pointers, as opposed to `[]nodes.Node`?

Reply:
This plays nicer with Delete(), so I'm on the fence about what to do here.

Reply:
SGTM. I don't think it's a big issue really 😄 

Feel free to `/hold cancel`!

Reply:
This actually annoys me but I haven't thought of a better solution yet so I'll just merge this for now 😅